src/app/_components/PortalComponents/Portal/recruitment/RecruitmentTable.tsx

```
@@ -14,9 +12,9 @@ export type RecruitmentForm = Pick<
 const RecruitmentTable = ({ forms }: { forms: RecruitmentForm[] }) => {
   return (
     <EntityTable
-      columns={columns as ColumnDef<RecruitmentForm, unknown>[]}
-      data={forms}
       tableHeader="Recruitment"
+      data={forms}
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)